smartcar.go

```diff
@@ -0,0 +1,64 @@
+// Package smartcar is the official Go SDK of the Smartcar API.
+// Smartcar is the only vehicle API built for developers, by developers.
+// Learn more about Smartcar here, https://smartcar.com/
+package smartcar
+
+import (
+	"time"
+
+	"github.com/smartcar/go-sdk/helpers/requests"
+	"github.com/smartcar/go-sdk/helpers/utils"
+)
+
+// GetUserID returns the user ID of the vehicle owner associated with the accessToken.
+func GetUserID(accessToken string) (string, error) {
```

Comment:
Several methods have a `Get` prefix (`GetUserID`, `GetVehicleIds`, ...), others don't (`vehicle.Info`, vehicle.VIN`). Can you make them consistent?